VocabularyTrainer/Language Screens/LanguageScreenViewController.swift

```diff
@@ -11,18 +11,153 @@ import MessageUI
-    @IBOutlet weak var newWordButton: UIButton!
-    @IBOutlet weak var deleteButton: UIButton!
-    @IBOutlet weak var exportButton: UIButton!
-    @IBOutlet weak var sortWordButton: UIButton!
-    @IBOutlet weak var sortTranslationButton: UIButton!
-    @IBOutlet weak var sortDateButton: UIButton!
-    @IBOutlet weak var hintLabel: UILabel!
-    
-    @IBOutlet weak var tableView: UITableView!
+    let buttonHeight: CGFloat = 36
+    let symbolConfig = UIImage.SymbolConfiguration(textStyle: .title1, scale: .large)
+
+    lazy var backButton: UIButton = {
+        let button: UIButton = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.heightAnchor.constraint(equalToConstant: buttonHeight).isActive = true
+        button.setTitle(NSLocalizedString("< Back", comment: "< Back"), for: .normal)
+        button.backgroundColor = BackgroundColor.hansaYellow
+        button.layer.cornerRadius = 5.0
+        button.contentEdgeInsets = UIEdgeInsets.init(top: 0, left: 10, bottom: 0, right: 10)
+        button.setTitleColor(BackgroundColor.japaneseIndigo, for: .normal)
+        return button
+    }()
+    
+    lazy var deleteButton: UIButton = {
+        let button: UIButton = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.heightAnchor.constraint(equalToConstant: buttonHeight).isActive = true
+        button.setTitle(NSLocalizedString("Delete Language", comment: "Delete Language"), for: .normal)
+        button.backgroundColor = BackgroundColor.red
+        button.layer.cornerRadius = 5.0
+        button.contentEdgeInsets = UIEdgeInsets.init(top: 0, left: 10, bottom: 0, right: 10)
+        button.setTitleColor(BackgroundColor.mediumWhite, for: .normal)
+        return button
+    }()
+    
+    lazy var searchBar: UISearchBar = {
+        let searchBar: UISearchBar = UISearchBar()
+        searchBar.translatesAutoresizingMaskIntoConstraints = false
+        searchBar.tintColor = UIColor(white: 1.0, alpha: 0.3)
+        return searchBar
+    }()
+    
+    lazy var tableView: UITableView = {
+        let tableView = UITableView(frame: CGRect(x: 0, y: 0, width: 0, height: 0))
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.backgroundColor = UIColor(white: 1.0, alpha: 0.3)
+        tableView.layer.cornerRadius = 10.0
+        return tableView
+    }()
+    
+    lazy var newWordButton: UIButton = {
+        let button: UIButton = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.titleLabel?.font = .systemFont(ofSize: 25)
+        button.setTitle(NSLocalizedString("New Word", comment: "New Word"), for: .normal)
+        button.backgroundColor = BackgroundColor.mediumSpringBud
+        button.layer.cornerRadius = 5.0
+        button.setTitleColor(BackgroundColor.japaneseIndigo, for: .normal)
+        return button
+    }()
+    
+    lazy var hintLabel: UILabel = {
```

Comment:
Can you check the constraints you did set for this?

It is not centered anymore:
![grafik](https://user-images.githubusercontent.com/35889530/198100076-d170e88d-bd03-4ab0-b1c7-c35dc956e2ea.png)

Other than that, great contribution! Thanks so much! :)

Reply:
Have fixed the alignment, Let me know if anything else is a blocker for the merge